src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -88,6 +88,7 @@
     private boolean enableRetransmissions = true;
     private int maximumPacketSize = 0;
     private String[] applicationProtocols = new String[0];
+    private String[] signatureSchemes = new String[0];
 
     /**
      * Constructs SSLParameters.
```

Comment:
Would it be useful to add another ctor that takes a signature schemes array parameter?

Reply:
As there is increasing number of SSL parameters, applications may just use the constructor with no argument, and use the set methods individually.

Reply:
Good catch.  Updated.

Reply:
Looks good.